clouds/snowflake/modules/test/h3/H3_TOCHILDREN.test.js

```diff
@@ -71,4 +71,4 @@ test('Coarser resolution returns empty array', async () => {
     const rows = await runQuery(query);
     expect(rows.length).toEqual(1);
     expect(rows[0].TOP_CHILDREN).toEqual([ ]);
-});
\ No newline at end of file
+});
```

Comment:
I see some spaces around. Please try `make lint` and then submit the results. We have pending adding a pre-commit to ensure this linting but it's a good practice to do it.

Reply:
Done. It turns out I didn't have the `virtualenv` command locally. So my linting was only partial. Fixed now.